cime_config/usermods_dirs/NEON/defaults/shell_commands

```diff
@@ -23,5 +24,13 @@ else
   ./xmlchange CLM_NML_USE_CASE="2018_control"
 fi
 
+# If needed for SP simulations: turn off lightning & set history file variables
+if  [[ $compset == *'SP_SICE'* ]]; then
+  ./xmlchange STOP_OPTION=nyears
```

Comment:
@wwieder the problem with explicitly setting STOP variables is that it conflicts with the ways either run_neon.py might set it (based on the --end-date option), or their own settings, or in my case my test cases.

Reply:
This will be discussed in #2036